elements/pfe-sass/maps/_typography.scss

```diff
@@ -0,0 +1,68 @@
+$pfe-typography-base: (
+  font-family:          '"Red Hat Text", "RedHatText", "Overpass", Overpass, Arial, sans-serif',
```

Comment:
Should we leave this since it already exists?  Technically in BEM this should be: `FontFamily` and `LineHeight`, etc.

Reply:
I didn't change these vars, just relocated them from [general.scss](https://github.com/patternfly/patternfly-elements/blob/master/elements/pfe-sass/maps/_general.scss#L20) over to _typography.scss for organization purposes. We could consider changing them to match PF core, but it would mean more breaking changes?

Reply:
PF core variables:
https://github.com/patternfly/patternfly/blob/master/src/patternfly/base/_variables.scss#L251